lib/beaker/ssh_connection.rb

```diff
@@ -30,7 +30,7 @@ def initialize name_hash, user = nil, ssh_opts = {}, options = {}
       @ip = name_hash[:ip]
       @hostname = name_hash[:hostname]
       @user = user
-      @ssh_opts = ssh_opts
+      @ssh_opts = ssh_opts.merge({keepalive: true, keepalive_interval: 2})
```

Comment:
Putting these settings here splits up the ssh settings (the others are in the [presets.rb](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/options/presets.rb#L191-L199)).  Please put these settings all in one place.